src/main/java/com/amplitude/Amplitude.java

```diff
@@ -96,6 +94,14 @@ public void setLogMode(AmplitudeLog.LogMode logMode) {
     this.logger.setLogMode(logMode);
   }
 
+  /**
+   * Set event callbacks which are triggered after event sent
+   * @param eventCallbacks
+   */
+  public void setEventCallbacks(List<AmplitudeEventCallback> eventCallbacks) {
```

Comment:
Why would we need more than one callback?

Reply:
This was required by doordash specifically. I can double check with them. My guess is for the callbacks, they want some flexibility or can have multiple callbacks from different services.

Reply:
ok. If we do accept a list of callbacks, we also need to have a func to take only one callback too.

Reply:
I see, do you mean add a method like `addEventCallback(AmplitudeEventCallback callback)` as an extra interface? And the one taken in will be added into current list? That could be an easier way then have to set the whole list each time.

Reply:
@haoliu-amp @justin-fiedler Just updated interface to `setEventCallback`